template/CHART_NAME/templates/role.yaml

```
@@ -1,6 +1,6 @@
 {{ if .Values.rbac.create }}
 kind: Role
-apiVersion: rbac.authorization.k8s.io/v1
+apiVersion:  {{ include "common.capabilities.rbac.apiVersion" . }}
```

Not to be picky, but could you please remove the extra whitespace?

```suggestion
apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
```